lib/Elastica/Query.php

```diff
@@ -118,21 +118,6 @@ public function getQuery()
     }
 
     /**
-     * Set Filter.
-     *
-     * @param \Elastica\Query\AbstractQuery $filter Filter object
-     *
-     * @return $this
-     *
-     * @link    https://github.com/elasticsearch/elasticsearch/issues/7422
-     * @deprecated Use Elastica\Query::setPostFilter() instead, this method will be removed in further Elastica releases
```

Comment:
setFilter has been deprecated long time ago,
have a look here #1138 
I've also had a look on a bunch of commits even inside this one : 47938554412886f61a3b87d655cb7bf78faee24b
which is from 21/07/2015 so it looks like this is very old